app/controllers/healthcheck_controller.rb

```diff
@@ -5,4 +5,13 @@ def overdue
     logger.error "HealthcheckController#overdue: #{e.message}"
     render json: { overdue: nil }, status: :service_unavailable
   end
+
+  def unenqueued_scheduled_editions
+    render json: {
+      unenqueued_scheduled_editions: Edition.future_scheduled_editions.count - ScheduledPublishingWorker.queue_size,
+    }
+  rescue ActiveRecord::StatementInvalid => e
```

Comment:
Since both controller methods catch this, is there a way we can do a [`rescue_from`](https://apidock.com/rails/ActiveSupport/Rescuable/ClassMethods/rescue_from) to reduce the duplication?

Reply:
(Can ignore, as this is more relevant to #6001)